data/json/bionics.json

```
@@ -1181,9 +1176,9 @@
   {
     "id": "bio_syringe",
     "type": "bionic",
-    "name": { "str": "Intravenous Needletip" },
-    "description": "The flesh of the inside of your middle finger has been replaced with a tube leading into your arteries, with a retractable needle extending out of the tip of the finger when clenched.  This lets you pump intravenous drugs like morphine directly into your bloodstream without needing to carry a syringe.",
-    "occupied_bodyparts": [ [ "hand_l", 1 ] ]
+    "name": { "str": "Intravenous Injector" },
+    "description": "You have a drug injector surgically mounted in your arm, with a retractable needle connected to your arteries.  This lets you pump intravenous drugs like morphine directly into your bloodstream without needing to carry a syringe.",
+    "occupied_bodyparts": [ [ "arm_l", 2 ] ] // Occupies 2 due to mechanism for cleaning needle
```

```suggestion
    "//": "Occupies 2 due to mechanism for cleaning needle",
    "occupied_bodyparts": [ [ "arm_l", 2 ] ]
```
JSON formatter was objecting...

---

Removed the comment altogether already

---

Ups, forgot to push